usaon_vta_survey/routes/home.py

```diff
@@ -1,6 +1,10 @@
+from flask import render_template
+
 from usaon_vta_survey import app
 
 
-@app.route("/home")
+@app.route("/")
 def root():
```

Comment:
I think we should name this file and the route consistently. Should we call this `root.py` or rename this function to `home`? If we rename the function, we also need to find any places where we do `url_for("root")` or similar, and update those. I think it's safest to update the filename, because our linter/typechecker can find any broken imports.

Reply:
went with changing the filename.